sdk/servicebus/service-bus/test/stress-test-track-2/scenarioBatchReceive.ts

```diff
@@ -106,7 +121,13 @@ export async function scenarioReceiveBatch() {
     }
   }
 
-  await Promise.all([sendMessages(), receiveMessages()]);
+  if (sendAllMessagesBeforeReceiveStarts) {
+    if (totalNumberOfMessagesToSend > ONE_MILLION) totalNumberOfMessagesToSend = ONE_MILLION;
```

Comment:
Just curious...why the 1 million message limit?

Reply:
Don't have a strong reason. Would you prefer not to have a limit? (I can switch accordingly)

Reply:
I think it'd be nice to be able to run with a higher number of messages unless there's a reason not to allow that.